app/javascript/components/Cell.js

```diff
@@ -1,29 +1,30 @@
-import React, { Component } from 'react'
-import AxiosInstance from '../actions/gameActions'
+import React, { Component } from "react";
+import AxiosInstance from "../actions/gameActions";
 
-import './game/game.css'
+import "./game/game.css";
 
-class Cell extends Component{
-    
-      onPlayHandler = () => { 
-        AxiosInstance({path: `games/${this.props.id}/move/${this.props.position}`, method: 'patch'})
-          .then(res => {                                    
-            this.props.playGame(res);
-          })
-      }
+class Cell extends Component {
+  onPlayHandler = () => {
+    AxiosInstance({
+      path: `games/${this.props.id}/move/${this.props.position}`,
+      method: "patch",
+    }).then((res) => {
+      this.props.playGame(res);
+    });
+  };
 
-    render(){      
-        return(
-            <li 
-            data-pos="0,0" 
-            onClick={() => this.onPlayHandler()} 
-            id="players" 
-            className={`${this.props.endGame ? "disableCell": ""}`} >
-              {this.props.message}
-            </li>
-         )
-    }
+  render() {
+    return (
+      <li
+        data-pos="0,0"
+        onClick={() => this.onPlayHandler()}
+        id="players"
+        className={`${this.props.endGame ? "disableCell" : ""}`}
```

Comment:
One common utility for constructing CSS class names in React is [clsx](https://www.npmjs.com/package/clsx), which could make the code a bit easier to read. For example, you could write:

```js
clsx(this.props.endGame && 'disableCell')
```